src/gfx/freeimage.cpp

```diff
@@ -252,7 +252,7 @@ bool GfxProcFreeImage::readbitmapFfmpeg(FileAccess* fa, const LocalPath& imagePa
 
     string extension;
     if (client->fsaccess->getextension(imagePath, extension)
-            && extension.compare(".mp3") && seek_target > 0
+            && strcmp(extension.c_str(),".mp3") && seek_target > 0
```

Comment:
I'm a bit confused about what git applied in this merge, or maybe this is a third version of achieving the same comparison, that is now in develop head? i.e. using c_str() with unsafe strcmp